paper/writeup.tex

```diff
@@ -1159,32 +1186,38 @@ \section{Discussion}
 of the true ancestry of a set of samples (i.e., all their genetic ancestors who ever lived),
 and so we wanted to quantify
 ``How much of the true relationships does this ARG represent?''
-However, the symmetrized version
+However, it is worth noting that the symmetrized version
 $d(\T_1,\T_2) = \dis(\T_1,\T_2) + \dis(\T_2, \T_1)$ is a metric.
 To see this, first suppose that we have a bijection between the nodes of $\T_1$ and $\T_2$,
 and view each ARG as a subset of the space $[0,L) \times N \times N$,
 where $N$ is the shared set of nodes.
-Then, dissimilarity is the Lesbegue measure of the relative difference of the two sets:
+Then, dissimilarity is the Lebesgue measure of the relative difference of the two sets:
 $|\T_1 \setminus \T_2|$,
 and so the symmetrized version is the measure of the symmetric difference
 $|\T_1\Delta\T_2|$,
-which is well-known to be a metric.
+which is well-known to be a metric. % NSP-22NOV: "well-known" ==> give a citation here
+% HF-24NOV: Peter would we cite Rudin?
 If the two ARGs have the same number of nodes,
-then we take the minimum over the all bijections between their nodes,
-and minima over a finite number of metrics is also a metric.
-This also holds for two tree sequences with different sizes of nodes, $|N_1|\neq|N_2|$ as we can take the minimum over all possible matchings $T_1\to\T_2$ and $\T_2\to\T_1$.
+we can consider all bijections between their nodes.
+The symmetric difference between $\T_1$ and $\T_2$,
+related through each bijection is a metric.
+Then the minimum over all such metrics will still be a metric
+since the minimum over a finite number of metrics is also a metric.
+ % NSP-22NOV: I don't understand this sentence, can you rewrite to be clearer?
+ % HF-24NOV: Hopefully it is clearer now
+This also extends to two tree sequences with different numbers of nodes, $|N_1|\neq|N_2|$, as we can take the minimum over all possible matchings $T_1\to\T_2$ and $\T_2\to\T_1$.
```

Comment:
I'm not quite following this - given two functions `f : [N_1] \to [N_2]` and `g : [N_2] \to [N_1]`, I think you're saying the size of the symmetric difference is a metric, but on what? The size of the symmetric difference is a mapping from (ARGs with N_1 nodes) \times (ARGs with N_2 nodes) to the nonnegative reals, but a metric needs to be on X \times X for some space X. Hm, I suppose you can just define all ARGs to be on countably many nodes, but most of which don't have any relationships between them. Ah, yes, this works, I think! And, a nice way to deal with this.

Reply:
Yeah if you consider the ambient space N_1 \times N_2 where both T_1 and T_2 are canonically embedded it just holds for the exact same reason the symmetric difference itself is a metric.